src/Sylius/Component/Core/Taxation/Applicator/OrderShipmentTaxesApplicator.php

```diff
@@ -61,12 +61,12 @@ public function apply(OrderInterface $order, ZoneInterface $zone): void
             /** @var TaxRateInterface|null $taxRate */
             $taxRate = $this->taxRateResolver->resolve($shippingMethod, ['zone' => $zone]);
             if (null === $taxRate) {
-                return;
+                continue;
             }
 
             $taxAmount = $this->calculator->calculate($shipment->getAdjustmentsTotal(), $taxRate);
             if (0.00 === $taxAmount) {
-                return;
+                continue;
```

Comment:
Do you think it makes sense to cover this case as well? I do not see it in spec

Reply:
To be honest, I don't think it makes sense, but of course, I can do that